docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,43 @@
+# Metadata
```

Comment:
@nawab-samsung Please don't forget to guide to update toc_all.md file when a new page is added.

Reply:
@pkosko 
1. Kindly correct the typo mistake of: 
#### [Ovierview](/application/web/guides/multimedia/overview.md)

Ovierview  --> Overview 

2. Kindly update the toc file.
https://github.com/Samsung/tizen-docs/blob/master/docs/application/toc_all.md


### Media and Camera
#### [Overview](/application/web/guides/multimedia/overview.md)
#### [JPEG File EXIF Information](/application/web/guides/multimedia/jpeg-exif.md)
#### [Audio Management](/application/web/guides/multimedia/audio.md)
#### [Audio Latency](/application/web/guides/multimedia/player-util.md)
#### [Media Controller](/application/web/guides/multimedia/media-controller.md)
#### [Media Key Events](/application/web/guides/multimedia/media-key.md)
#### [Radio](/application/web/guides/multimedia/radio.md)
#### [Camera](/application/web/guides/multimedia/camera.md)
#### [Metadata](/application/web/guides/multimedia/metadata.md)